src/js/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -37,7 +37,7 @@ class DocumentRequestPage extends React.Component {
     if (this.props.trackedItem) {
       document.title = `Request for ${this.props.trackedItem.displayName}`;
     } else {
-      document.title = 'Document Request';
+      this.props.router.push(`/your-claims/${this.props.params.id}/status`);
```

Comment:
I don't think this is the right place for redirecting. This method is only called once, after rendering, so it probably won't avoid the error we're seeing. We also need to wait for claim information to load, before we know for sure that the tracked item is missing.

I think you want to put this in `componentWillReceiveProps` and redirect when the `loading` prop becomes true and there's no `trackedItem`.

Reply:
Thanks for pointing that out @jbalboni, change made.